src/main/java/spireMapOverhaul/zones/brokenSpace/cardmods/UnreadableCardMod.java

```diff
@@ -0,0 +1,69 @@
+package spireMapOverhaul.zones.brokenSpace.cardmods;
+
+import basemod.abstracts.AbstractCardModifier;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+
+import static spireMapOverhaul.SpireAnniversary6Mod.makeID;
+
+public class UnreadableCardMod extends AbstractCardModifier {
+    public static final String ID = makeID("UnreadableCardMod");
+    private String name;
+
+    @Override
+    public void onInitialApplication(AbstractCard card) {
+
+        name = randomText(card.name.length());
+        card.exhaust = true;
+        card.isEthereal = true;
+        card.purgeOnUse = true;
```

Comment:
You shouldn't use the `purgeOnUse` field. It has strange interactions, such as causing the card to not trigger Burst/Echo Form/etc.

Instead, you can use StsLib's purge modifier: `PurgeField.purge.set(card, true)`.

Also, if you set the card to purge, having it exhaust is redundant; purge overrides exhaust (the card will not go to the exhaust pile and it will not trigger effects like Feel No Pain). You can get rid of the exhaust part of this and change the relic description to say that it purges.